Week4/app/src/main/res/layout/activity_movie_detail.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
You can swap this for `FrameLayout` just to be a bit more efficient. Frame is lighter than Constraint